README.md

```diff
@@ -370,6 +370,8 @@ Tests will detect it and run a set of integration tests in a new namespace.
 
 Some integration tests run a local jenkins, so the host that runs them needs
 to be accessible from the kubernetes cluster.
+By default Jenkins will listen on `192.168.64.1` interface only, for security reasons.
```

Comment:
Is it not 192.168.99.1 ?

Reply:
in my machine is 192.168.64.1 so I guess it may be different in each installation

Reply:
hmm...in minikube start help:

`--host-only-cidr string           The CIDR to be used for the minikube VM (only supported with Virtualbox driver) (default "192.168.99.1/24")`

If it is different for each installation, then many ppl will get bit by not passing the right connector property.

Reply:
You can get the IP via `minikube ip` on the host; maybe you can use a Makefile or something to get the value of `minikube ip` and pass it as a parameter to tests or something?